docs/reference/requirement-sets/excel-api-requirement-sets.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Excel JavaScript API requirement sets
 description: 'Office Add-in requirement set information for Excel builds'
-ms.date: 07/11/2019
+ms.date: 07/17/2019
```

Comment:
Suggest that you not update ms.date in this article, as the changes have no substantive affect on the article's content; we don't want to give readers the impression that something has changed in the article when, in fact, nothing has really changed.